cmd/keytransparency-signer/backend.go

```diff
@@ -97,10 +124,22 @@ func main() {
 		log.Fatalf("Failed to create SQL history: %v", err)
 	}
 	mutator := entry.New()
-	sths, err := appender.New(context.Background(), sqldb, *mapID, *mapLogURL, nil)
+
+	// STH Appender.
+	// The log should be in a restricted access backend environment.
+	cc, err := grpc.Dial(*logURL, grpc.WithInsecure())
```

Comment:
Great point. Trillian does not not current support HTTPS connections. 
I raised this question in https://github.com/google/trillian/issues/428 

I think this is because Trillian is not designed to ever be a front end / exposed service. It also lacks other front end type things like authentication, quota, and anti-abuse mechanisms.